checkstyle-sonar-plugin/src/main/resources/org/sonar/l10n/checkstyle.properties

```diff
@@ -177,7 +177,6 @@ rule.checkstyle.com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocStyleCheck
 rule.checkstyle.com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocStyleCheck.param.checkFirstSentence=Whether to check the first sentence for proper end of sentence. Default is true.
 rule.checkstyle.com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocStyleCheck.param.endOfSentenceFormat=Format for matching the end of a sentence.
 rule.checkstyle.com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck.name=Line Length
-rule.checkstyle.com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck.param.tabWidth=number of expanded spaces for a tab character (''\t'')
```

Comment:
please share a screenshot of how `tabWidth` is looks like in UI for this Check, I think we are loosing documentation content by removal of this line

Reply:
configuration of this check:
![linelength_config](https://user-images.githubusercontent.com/653739/56283358-cf31da80-6111-11e9-8302-7f844192355b.PNG)

rule violation in project:
![linelength_project](https://user-images.githubusercontent.com/653739/56283360-cf31da80-6111-11e9-814a-15a701836996.PNG)

Reply:
hmm, so it becomes not configurable , but it was.
Can we keep it? It should not hurt. User can override Checker's tabWidth, most users will not, but it give some flexibility to user to have few instances of Checks with different tabWidth and config and reconcile overlap/extra violations by suppressions.

Please share a screenshot of tabWidth in Checker,  my original question was to make sure that tabWidth in Checker have some hint/note like `number of expanded spaces for a tab character (''\t'')`

Reply:
Just saw this. From a user's perspective, I should hope that my tabs have equal width in the eyes of all checks. Adding the possibility of having different tab widths per check adds a source of errors. (Per *file* might make some sense, but per *check* is confusing.)

Reply:
I agree with @tsjensen and @rnveach proposed the deletion of tabWidth in those checks as well (https://github.com/checkstyle/sonar-checkstyle/issues/206#issuecomment-482656114).
Does the user really need to set a different tabWidth for those checks and what use cases does that cover where tabWidths are intentionally different?

No, I just referred to the documentation of this property in CS (per link). I‘ll add this sentence for clarification to the user - PR update comes tomorrow. :)

Reply:
@romani 
> We do not add tabWidth to all other Checks (for now), but lets keep smth that users requested already.

Based on your conversations before, I was going to start a breaking issue in main repo and remove tabWidth setter from everything except Checker so it is no longer a global property. This would align with how this PR is currently handling tabWidth. The reasoning was if we were against tabWidth in every check here, why would we allow main repo to do it.

I will wait until I see how we resolve this thread.

Reply:
I added the note to the description now. Please advice which direction regarding `tabWidth` we'll be going.

Reply:
Please leave such properties in place in this release. Right now it is possible in library to define checker  and checks property to be different. If in future it might change, but we do know when, so we will adjust plugin accordingly.

Let's move this discussion to library project, it is not easy issue.
Plugin release should not be blocked by this.

FYI, sonarqube plugin does not critically depends on correct column reporting. Such precise position of violation is more in demand in IDE plugins, where user is positioned to exact column for fix, or used to make a fix automatically.

Reply:
OK, reverted.